### DISCUSSION TITLE
Issue with multiple Tooltips

### DISCUSSION THREAD

**Discussion by Elendiar** [ant-design/ant-design#41583] [Q&A]
A lot of tooltips are the cause of a significant decrease of perfomance. In my app, (keybox, with a lot of position and draggable-dnd keys components) when wrapped with Tooltip, on key drag app starts to slow down and freeze with every tooltip rerendered.
Is in antd way to optimize Tooltip usage with many (~100) components? Something like static message/notification/modal, when used one top-level tooltip for all components? Is there a better way, than render Tooltip only onMouseEnter?
![image](https://user-images.githubusercontent.com/3886001/229270490-3c6ec195-3cdc-464f-86b3-39896e32e2db.png)

**Comment by MbIWaTUK** [comment]
I have the same problem
This problem only exists on version 5
